apps/ledger-live-desktop/src/internal/commandHandler.js

```diff
@@ -33,8 +45,10 @@ export function executeCommand(command: *, send: *) {
   const startTime = Date.now();
   logger.onCmd("cmd.START", id, 0, data);
   try {
-    subscriptions[requestId] = cmd(data).subscribe({
+    subscriptions[requestId] = cmd(data); // Store the subject returned instead of the subscription
+    subscriptions[requestId].subscribe({
       next: data => {
+        if (data?.type === "input-frame") return; // TODO Input can't be treated as output too
```

Comment:
not ideal to hack some knowledge of the commands in the generic rpc logic. let's figure out something